src/main/java/li/cil/oc2/common/blockentity/ChargerBlockEntity.java

```diff
@@ -153,4 +153,12 @@ private void charge(final IEnergyStorage energyStorage) {
             isCharging = true;
         }
     }
+
+    @Override
+    public AABB getRenderBoundingBox() {
+        return new AABB(
```

Comment:
Recalculating this on every call (every frame for each `BlockEntity`) is redundant, since the `BlockPos` of a `BlockEntity` is final. Instead, this should be calculated once in the constructor and stored in a field.

The only reason Forge doesn't do this in the default implementation is to avoid patching `BlockEntity`, which would be necessary since interfaces don't allow non-static fields.